### PR TITLE
pla.nette.org translation/missing translated

### PR DESCRIPTION
**Pull request by meridius**
This added mostly already translated articles from play.nette.org, which were not yet part of the codebase.

They are located in temp directory and are missing timestamp from their filenames and from "details" tables because date on play.nette.org is not exact.   
@paveljanda could you please send me those dates?

I've also created new overview table reflecting those changes https://docs.google.com/spreadsheets/d/1e57AO2FXDKZXE5nhnMPWI2jtPaYaAQ_8hx-Z7JFm-go/edit?usp=sharing

### CODE REVIEW

**Comment by f3l1x** [comment]
Look's good. Do you still working on that?

**Comment by meridius** [comment]
Just waiting on those dates.

File: content/oldPlanette/x/connect-dibi-as-a-service-with-tracy-bar.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+date: ""
```
Comment (paveljanda) [inline content/oldPlanette/x/connect-dibi-as-a-service-with-tracy-bar.md]: 2014-10-28 11:27:11

File: content/oldPlanette/x/dibi-fluent-insert.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+date: ""
```
Comment (paveljanda) [inline content/oldPlanette/x/dibi-fluent-insert.md]: 2014-10-28 10:19:14

File: content/oldPlanette/x/different-layout-in-administration.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
+---
+date: ""
```
Comment (paveljanda) [inline content/oldPlanette/x/different-layout-in-administration.md]: 2015-04-24 20:12:32

File: content/oldPlanette/x/examples-of-macro-n-class-usage.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+date: ""
```
Comment (paveljanda) [inline content/oldPlanette/x/examples-of-macro-n-class-usage.md]: 2014-10-28 11:36:56

File: content/oldPlanette/x/form-image-and-rules.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+---
+date: ""
```
Comment (paveljanda) [inline content/oldPlanette/x/form-image-and-rules.md]: 2014-10-28 10:35:36

File: content/oldPlanette/x/how-to-ajax-response-with-json.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
+---
+date: ""
```
Comment (paveljanda) [inline content/oldPlanette/x/how-to-ajax-response-with-json.md]: 2014-11-19 00:46:00

File: content/oldPlanette/x/how-to-change-latte-syntax-for-javascript.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
+---
+date: ""
```
Comment (paveljanda) [inline content/oldPlanette/x/how-to-change-latte-syntax-for-javascript.md]: 2014-10-28 11:22:57

File: content/oldPlanette/x/how-to-setup-simpleauthenticator.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
+---
+date: ""
```
Comment (paveljanda) [inline content/oldPlanette/x/how-to-setup-simpleauthenticator.md]: 2014-11-02 11:34:40

File: content/oldPlanette/x/how-to-test-exception-call.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
```
Comment (paveljanda) [inline content/oldPlanette/x/how-to-test-exception-call.md]: 2014-10-28 18:21:20

File: content/oldPlanette/x/how-to-use-texy-in-latte-template-as-a-filter.md [hunk 10]
```diff
@@ -0,0 +1,39 @@
+---
+date: ""
```
Comment (paveljanda) [inline content/oldPlanette/x/how-to-use-texy-in-latte-template-as-a-filter.md]: 2015-02-01 16:59:53

File: content/oldPlanette/x/howto-lazy-factory.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
+---
+date: ""
+draft: false
+title: "Howto Lazy Factory"
+tags: ["snippet", "dependency injection"]
+type: "blog"
+slug: "howto-lazy-factory"
```
Comment (paveljanda) [inline content/oldPlanette/x/howto-lazy-factory.md]: Maybe "ho to CREATE lazy factory"?

File: content/oldPlanette/x/howto-lazy-factory.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
+---
+date: ""
```
Comment (paveljanda) [inline content/oldPlanette/x/howto-lazy-factory.md]: 2014-10-28 11:40:28

File: content/oldPlanette/x/latte-simple-foreach-with-first-last-macro.md [hunk 13]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
```
Comment (paveljanda) [inline content/oldPlanette/x/latte-simple-foreach-with-first-last-macro.md]: 2014-10-28 11:23:27

File: content/oldPlanette/x/overwrite-whole-section-in-neon-config.md [hunk 14]
```diff
@@ -0,0 +1,25 @@
+---
+date: ""
```
Comment (paveljanda) [inline content/oldPlanette/x/overwrite-whole-section-in-neon-config.md]: 2014-10-28 12:21:55

File: content/oldPlanette/x/recursive-delete-folder.md [hunk 15]
```diff
@@ -0,0 +1,21 @@
+---
+date: ""
```
Comment (paveljanda) [inline content/oldPlanette/x/recursive-delete-folder.md]: 2014-10-28 11:24:06

File: content/oldPlanette/x/recursive-delete-folder.md [hunk 16]
```diff
@@ -0,0 +1,21 @@
...
+draft: false
+title: "Recursive delete folder"
```
Comment (paveljanda) [inline content/oldPlanette/x/recursive-delete-folder.md]: RecursiveLY? :)
Or Recursive directory deleting

File: content/oldPlanette/x/simple-way-hot-to-setup-basic-router-from-config-file.md [hunk 17]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
```
Comment (paveljanda) [inline content/oldPlanette/x/simple-way-hot-to-setup-basic-router-from-config-file.md]: 2014-10-28 12:23:25

File: content/oldPlanette/x/use-persistent-parameters-in-presenters.md [hunk 18]
```diff
@@ -0,0 +1,18 @@
+---
+date: ""
```
Comment (paveljanda) [inline content/oldPlanette/x/use-persistent-parameters-in-presenters.md]: 2014-10-28 12:26:24

File: content/oldPlanette/x/conditional-button-in-form.md [hunk 19]
```diff
@@ -0,0 +1,40 @@
+---
+date: ""
```
Comment (paveljanda) [inline content/oldPlanette/x/conditional-button-in-form.md]: 2014-11-05 20:31:41

**Comment by paveljanda** [comment]
@meridius Added dates + 2 notes :)

**Comment by meridius** [comment]
@paveljanda cool! Where do I find that? 😄

**Comment by paveljanda** [comment]
@meridius 
Sorry! Didn't submit the review..

**Review by paveljanda — CHANGES_REQUESTED** [review]

**Comment by f3l1x** [comment]
@meridius We have a dates! :-) @paveljanda didn't submit his review :-D Lol!

**Comment by f3l1x** [comment]
download

**Comment by meridius** [comment]
@f3l1x any idea of what's wrong? It doesn't make sense to me.
https://travis-ci.org/planette/planette/builds/254230384#L286

Thanks

**Comment by f3l1x** [comment]
@meridius 

```
ERROR 2017/07/16 20:22:37 failed to parse page metadata for "oldPlanette/2014/2014-10-28-examples-of-macro-n-class-usage.md": yaml: line 3: did not find expected key
Error: Error building site: Errors reading pages: Error: failed to parse page metadata for "oldPlanette/2014/2014-10-28-examples-of-macro-n-class-usage.md": yaml: line 3: did not find expected key for 2014-10-28-examples-of-macro-n-class-usage.md
```

It looks like some typo, spacing or missing key?

File: content/oldPlanette/2014/2014-10-28-examples-of-macro-n-class-usage.md [hunk 20]
```diff
@@ -0,0 +1,23 @@
+---
+date: "2014-10-28"
+draft: false
+title: "Examples of macro "n:class" usage"
```
Comment (meridius) [inline content/oldPlanette/2014/2014-10-28-examples-of-macro-n-class-usage.md]: Note to self: Fix quotes within quotes.

**Comment by meridius** [comment]
I was blind but now I see. 😂

File: content/oldPlanette/2014/2014-10-28-examples-of-macro-n-class-usage.md (hunk 20 above)
Comment (f3l1x) [reply content/oldPlanette/2014/2014-10-28-examples-of-macro-n-class-usage.md]: `"Examples of macro \"n:class\" usage"`, doesn't work?

**Comment by meridius** [comment]
I think, this can be merged now.

**Comment by paveljanda** [comment]
Should be the tag name `n:macro` or just `macro`? People would have to distinguish between them which can be misleading..

**Comment by meridius** [comment]
Well, the snippet is for `n:class`, which is a `n:macro`. So it makes sense to me that it should have the same named tag. Naming it just `macro` could confuse readers that the article is about bracket `{macros}`.

**Comment by paveljanda** [comment]
`n:macro` is just a special case of `macro`, hm? :) :)
Viz https://latte.nette.org/cs/macros

**Comment by meridius** [comment]
OK. So how about:

```
title: "Examples of macro \"n:class\" usage"
tags: ["snippet", "latte", "macro"]
```

**Comment by paveljanda** [comment]
Love it. Thanks a lot and sorry for me being so pedantic. :D

**Comment by meridius** [comment]
No problem. I'm the same way. ;)

**Comment by f3l1x** [comment]
@meridius Big thanks! :+1: 

Also pedantic @paveljanda, you :dog2:!

**Pull request merged**